packages/cap-config-writer/src/cap-config/index.ts

```diff
@@ -0,0 +1,55 @@
+import { join } from 'path';
+import { create as createStorage } from 'mem-fs';
+import { create } from 'mem-fs-editor';
+import type { Editor } from 'mem-fs-editor';
+import type { Package } from '@sap-ux/project-access';
+import {
+    addCdsPluginUi5,
+    enableWorkspaces,
+    ensureMinCdsVersion,
+    getWorkspaceInfo,
+    hasCdsPluginUi5,
+    hasMinCdsVersion
+} from './package-json';
+
+/**
+ * Enable workspace and cds-plugin-ui5 for given CAP project.
+ *
+ * @param basePath - root path of the CAP project, where package.json is located
+ * @param [fs] - optional: the memfs editor instance
+ * @returns Promise<Editor> - memfs editor instance with updated files
+ */
+export async function enabledCdsUi5Plugin(basePath: string, fs?: Editor): Promise<Editor> {
```

Comment:
Should this be named `enableCdsUi5Plugin` rather than `enabledCdsUi5Plugin`?

Reply:
Good catch, fixed it